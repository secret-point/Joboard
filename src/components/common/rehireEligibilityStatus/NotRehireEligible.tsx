import React from "react";
import { Col } from "@amzn/stencil-react-components/layout";
import { Text } from "@amzn/stencil-react-components/text";
import { translate as t } from "../../../utils/translator";

const NotRehireEligible = () => {
  return (
    <Col gridGap="S300">
      <Text>
        {t("BB-not-rehire-eligible-title-text", "Thank you for your recent job application to Amazon. After careful consideration, we will not be moving forward with your job application.")}
      </Text>
      <Text>
        {t("BB-not-rehire-eligible-content-text", "We value your time and wish you the best of luck in the future.")}
      </Text>
    </Col>
  );
};

export default NotRehireEligible;
