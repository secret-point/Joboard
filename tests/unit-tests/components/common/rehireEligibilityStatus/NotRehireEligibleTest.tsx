import React from "react";
import { shallow } from "enzyme";
import NotRehireEligible from "../../../../../src/components/common/rehireEligibilityStatus/Days365NotRehireEligible";


describe("NotRehireEligible", () => {
    it("should match snapshot", () => {
        const shallowWrapper = shallow(<NotRehireEligible/>);
    
        expect(shallowWrapper).toMatchSnapshot();
    });
});