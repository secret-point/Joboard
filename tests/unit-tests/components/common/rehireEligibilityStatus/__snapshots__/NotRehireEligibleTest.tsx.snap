// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotRehireEligible should match snapshot 1`] = `
<Col
  gridGap="S300"
>
  <Text
    font="primary"
    fontSize="T300"
    hyphens="none"
  >
    Thank you for your recent job application to Amazon. After careful consideration, we will not be moving forward with your job application.
  </Text>
  <Text
    font="primary"
    fontSize="T300"
    hyphens="none"
  >
    We value your time and wish you the best of luck in the future.
  </Text>
</Col>
`;
